docs/skopeo-copy.1.md

```diff
@@ -42,57 +46,119 @@ Path of the authentication file for the source registry. Uses path given by `--a
 
 Path of the authentication file for the destination registry. Uses path given by `--authfile`, if not provided.
 
+**--dest-shared-blob-dir** _directory_
+
+Directory to use to share blobs across OCI repositories.
+
 **--digestfile** _path_
 
 After copying the image, write the digest of the resulting image to the file.
 
-**--format, -f** _manifest-type_ MANIFEST TYPE (oci, v2s1, or v2s2) to use in the destination (default is manifest type of source, with fallbacks)
+**--encrypt-layer** _ints_
+
+*(Experimental)*: key with the encryption protocol to use needed to encrypt the image (e.g. jwe:/path/to/key.pem)
```

Comment:
```suggestion
*Experimental* the 0-indexed layer indices, with support for negative indexing (e.g. 0 is the first layer, -1 is the last layer)
```
(I haven’t reviewed the details of the man pages any further for now.)

Reply:
Thank you, this is an important part of the feedback I'm looking for. I just copy-pasted the `--help` output.

Reply:
Copy&pasting `--help` is absolutely fair, this PR should not be blocked on agonizing over details of the full man page set.

I noted the above because the “0-indexed layer indices” text above is what I see in the `skopeo copy --help` output, so I was wondering if it suggests a bug in some automation used to generate the new text.